logback.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration debug="false" scan="true" scanPeriod="60 seconds">
+    <jmxConfigurator />
```

Comment:
Why do you need your own version of logback config? talk to Vikram

Reply:
removed this file